_apidocs/get-opportunities-public-api.md

```diff
@@ -778,6 +791,7 @@ Note: Response for one record is provided as an example <br>
             "typeOfSetAside": null,
             "responseDeadLine": null,
             "naicsCode": null,
+			"naicsCodes": [],
```

Comment:
There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?

Reply:
resolved